Box.V2/Models/BoxGroupFileCollaborationEventSource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+
+namespace Box.V2.Models
+{
+    public class BoxGroupFileCollaborationEventSource : BoxEntity
+    {
+        public const string FieldFileId = "file_id";
+        public const string FieldFileName = "file_name";
+        public const string FieldGroupId = "user_id";
+        public const string FieldGroupName = "user_name";
+        public const string FieldParent = "parent";
+
+        [JsonProperty(PropertyName = FieldFileId)]
```

Comment:
You shouldn't need `new` on any of these properties, since they doesn't collide with any properties in the base class.